packages/govuk-frontend/tasks/build/package.test.mjs

```diff
@@ -164,7 +164,7 @@ describe('packages/govuk-frontend/dist/', () => {
         const contents = await readFile(join(paths.package, 'dist/govuk/common/govuk-frontend-version.mjs'), 'utf8')
 
         // Look for ES modules `version` named export
-        expect(contents).toContain(`const version = '${pkg.version}';`)
+        expect(contents).toContain(`var version = '${pkg.version}';`)
```

Comment:
There's a comment mentioning that [Rollup might avoid this issue already](https://github.com/alphagov/govuk-frontend/pull/3728#discussion_r1251103256).

Is this a potentially temporary change before we check that?

Reply:
Yeah potentially. It's not causing any harm, but we should review and test impact before release

Reply:
Problem solved 🙌

I've enabled [`{ bugfixes: true }`](https://babeljs.io/docs/babel-preset-env#bugfixes)

It lets Babel use smaller transforms where browser features are only partially supported

For example, Babel knows that full support for [**template strings** as “tagged templates with escape sequences”](https://caniuse.com/mdn-javascript_grammar_template_literals_template_literal_revision) only shipped in Safari 11 but it lets us use plain old [**template strings**](https://caniuse.com/template-literals) right back to Safari 9.1 too

Thank goodness, because we [only recently switched to using them](https://github.com/alphagov/govuk-frontend/commit/0305cf6b42fae8e46ab4e180028308e290980bcf) 😣